content/kubermatic/main/tutorials-howtos/gitops-argocd/_index.en.md

```diff
@@ -0,0 +1,241 @@
+    1. Promtail
+    1. Loki
+1. S3 like object storage - minio
+1. User-cluster mla components
+    1. Minio and Minio Lifecycle Manager
+    1. Grafana
+    1. Consul
+    1. Cortex
+    1. LokiDistributed
+    1. AlertmanagerProxy
+    1. iap for user-mla
+    1. secrets - grafana and minio secrets
+1. Seed Settings - Kubermatic configuration, Seed objects, preset objects and such misc objects needed for Seed configuration
+1. Seed Extras - This is a generic ArgoCD app to deploy arbitrary resources not covered by above things and as per needs of KKP Admin.
+
+## Installation
+
+> You can find same tutorial code with sample values in [this git repository](https://github.com/dharapvj/kkp-using-argocd). For ease of installation, a `Makefile` has been provided to just make commands easier to read. Internally, it just depends on helm, kubectl and kubermatic-installer binaries. But you will need to look at `make` target definitions in `Makefile` to adjust DNS names. While for the demo, provided files would work, you would need to look through each file under `dev` folder and customize the values as per your need.
+
+### Setup two Kubernetes Clusters
+> This step install two Kubernetes clusters using Kubeone in AWS. You can skip this step, if you already have access to two kubernetes clusters.
+
+Use kubeone to create 2 clusters in DEV env - master-seed combo (c1) and regular seed (c2). Steps below are generic to any kubeone installation. a) We create basic VMs in AWS using terraform and then b) Use kubeone to bootstrap the control plane on these VMs and then rollout worker node machines.
+
+**Note:** The sample code provided here to create kubernetes clusters uses single VM control-plane. This is NOT recommended in any way as production. Always use HA control-plane for any production grade kubernetes installation.
+
+You should be looking at `terraform.tfvars` and `kubeone.yaml` files to customize these folder as per your needs.
+
+```shell
+# directory structure
+kubeone-install
+├── dev-master
+│   ├── kubeone.yaml
+│   ├── main.tf
+│   ├── output.tf
+│   ├── terraform.tfvars
+│   ├── variables.tf
+│   ├── versions.tf
+├── dev-seed
+│   ├── kubeone.yaml
+│   ├── main.tf
+│   ├── output.tf
+│   ├── terraform.tfvars
+│   ├── variables.tf
+│   ├── versions.tf
+└── kubeone
+
+# Kubeone needs ssh-keys to communicate with machines.
+# make sure to use ssh-keys which do not need any passphrase 
+
+# Setup CP machines and other infra
+export AWS_PROFILE=<your AWS profile>
+make k1-apply-master
+make k1-apply-seed
+```
+
+This same folder structure can be further expanded to add kubeone installations for additional environments like staging and prod.
+
+### Note about URLs:
+The [demo codebase](https://github.com/dharapvj/kkp-using-argocds) assumes `argodemo.lab.kubermatic.io` as base URL for KKP. KKP Dashboard is available at this URL. So master argocd, all master tools like prometheus, grafana, etc are accessible at `*.argodemo.lab.kubermatic.io`
```

Comment:
* Should this repo be moved?
* _The_ KKP dashboard
* "So master argocd"?
* Prometheus, Grafana capitalized

Reply:
> Should this repo be moved?

I also would like it to be under `Kubermatic` umbrella. Lets talk about where can we place it an I would be happy to move.

Reply:
I cannot judge this. If @moadqassem had no complaints, then we can keep it there.